tests/test_utils/test_internetspeed.py

```diff
@@ -30,12 +30,12 @@ class TestInternetSpeed:
     """
 
     def test_measurespeed_invalid_url(self):
-        speed = measurespeed("www.fake-url-9999999.xyz")
+        speed = measure_speed_from_url("www.fake-url-9999999.xyz")
```

Comment:
While you're at it, consider replacing this url with one in a reserved top-level domain (rfc6761 suggests .invalid, .example, .test would be good choices).